lms/templates/login.html

```diff
@@ -130,6 +130,8 @@
   </script>
 </%block>
 
+<%include file="forgot_password_modal.html" />
```

Comment:
This makes sense to me. I can't picture how you would ever reach the forgot password modal any other way.